packages/server/graphql/handleGraphQLTrebuchetRequest.ts

```diff
@@ -63,7 +65,7 @@ const handleGraphQLTrebuchetRequest = async (
             docId: docId || '',
             query: query || '',
             variables: JSON.stringify(variables),
-            socketServerId: socketId
+            socketServerId: SERVER_ID!
```

Comment:
I think it would also be nice to know which server timed out, i.e. the socketId like you had before.

Reply:
that's what it does now, the `socketId` was erroneously referring to the connectionContext.id. We could add both though! but that `socketId` goes away whenever someone leaves the app so there's not much to learn from it